Example/Backpack/UIKit/Components/Card/Card Button/CardButtonsViewController.swift

```diff
@@ -0,0 +1,280 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+import Backpack
+
+private struct CardButtonSampleRow {
+    let title: String
+    let cardButtons: [CardButtonSample]
+}
+
+private struct CardButtonSample {
```

Comment:
nit: consider moving this inside the view controller so that's clear that its not meant to be used elsewhere (if so, names can be less verbose!)

Reply:
13fd5bfed2e7093e80ce6e150a0d38f827068c68